spec/index.bs

```diff
@@ -410,6 +410,23 @@ The [=subresources=] of textures included in the views provided to
 are considered to have {{GPUTextureUsage/OUTPUT_ATTACHMENT}}
 for the [=usage scope=] of this render pass.
 
+The <dfn dfn>physical size</dfn> of a {{GPUTexture}} [=subresource=] is the dimension of the {{GPUTexture}}
+[=subresource=] in texels that includes the possible extra paddings to form complete [=texel blocks=] in the
+[=subresource=].
+
+  - For pixel-based {{GPUTextureFormat}}s, the [=physical size=] is always equal to the size of the [=subresource=]
+    used in the sampling hardwares.
+  - {{GPUTexture}}s in compressed {{GPUTextureFormat}}s always have a [=mipmap level=] 0 whose {{GPUTexture/[[textureSize]]}}
+    is a multiple of the [=texel block size=], but the lower mipmap levels might not be the multiple of the [=texel block size=]
+    and can have paddings.
+
+<div class="example">
+Considering a {{GPUTexture}} in BC format whose {{GPUTexture/[[textureSize]]}} is {60, 60, 1}, when sampling
+the {{GPUTexture}} at [=mipmap level=] 2, the sampling hardware uses {15, 15, 1} as the size of the [=subresource=],
+while its [=physical size=] is {16, 16, 1} as the block-compression algorithm can only operate on 4x4 [=texel blocks=].
+There is no extra padding for the {{GPUTexture}} [=subresources=] with non-compressed {{GPUTextureFormat}}s.
```

Comment:
This sentence is redundant with the pixel-based format list item above. I think it could be removed.

Reply:
Done